diff --git a/node_modules/remix-auth-oauth2/build/index.js b/node_modules/remix-auth-oauth2/build/index.js
index 4d8335b..bcba1b4 100644
--- a/node_modules/remix-auth-oauth2/build/index.js
+++ b/node_modules/remix-auth-oauth2/build/index.js
@@ -80,7 +80,7 @@ class OAuth2Strategy extends remix_auth_1.Strategy {
         // Redirect the user to the callback URL
         if (url.pathname !== callbackURL.pathname) {
             debug("Redirecting to callback URL");
-            let state = this.generateState();
+            let state = this.generateState(options.state);
             debug("State", state);
             session.set(this.sessionStateKey, state);
             throw (0, server_runtime_1.redirect)(this.getAuthorizationURL(request, state).toString(), {
@@ -206,8 +206,9 @@ class OAuth2Strategy extends remix_auth_1.Strategy {
         url.search = params.toString();
         return url;
     }
-    generateState() {
-        return (0, uuid_1.v4)();
+    generateState(state) {
+        const str = JSON.stringify({ nonce: (0, uuid_1.v4)(), ...state });
+        return Buffer.from(str).toString("base64");
     }
     /**
      * Format the data to be sent in the request body to the token endpoint.
